node/docs/genmd.js

```diff
@@ -102,6 +102,8 @@ writeLine();
 writeLine('The build generates a vsts-task-lib.d.ts file for use when compiling tasks');
 writeLine('In the example below, it is in a folder named definitions above the tasks lib');
 writeLine();
+writeLine('## [Release notes](releases.md)');
```

Comment:
gendocs calls into typedoc which hasn't caught up to ts compiler we need.  I've been hand editing docs since.  You're not changing signature - just impl so no need for docs update unless you want to directly update the comments